easybuild/easyconfigs/u/uncertainty-calibration/uncertainty-calibration-0.0.9-foss-2021b.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'PythonBundle'
+
+name = 'uncertainty-calibration'
+version = '0.0.9'
+
+homepage = 'https://github.com/p-lambda/verified_calibration'
+description = """Python library to measure the calibration error of models, including confidence intervals computed by
+Bootstrap resampling, and code to recalibrate models."""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('SciPy-bundle', '2021.10'),
+    ('scikit-learn', '1.0.1'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('sklearn', '0.0', {
```

Comment:
I dislike this and I am wondering if #15613 would be a better solution.

Reply:
Oh yes, I think the approach in #15613 makes sense...

Reply:
fixed in 2b62347